h/config.py

```diff
@@ -60,6 +62,10 @@
     EnvSetting('nsq.reader.addresses', 'NSQ_READER_ADDRS'),
     EnvSetting('nsq.writer.address', 'NSQ_WRITER_ADDR'),
     EnvSetting('origins', 'ALLOWED_ORIGINS'),
+    EnvSetting('rabbitmq.address', 'RABBITMQ_ADDRESS'),
+    EnvSetting('rabbitmq.user', 'RABBITMQ_USER'),
+    EnvSetting('rabbitmq.password', 'RABBITMQ_PASSWORD'),
+    EnvSetting('rabbitmq.vhost', 'RABBITMQ_VHOST'),
```

Comment:
As discussed in Slack: Supporting docker links auto-magically is increasingly becoming a problem. It is not very easy to support docker links with services that are not set up with default configuration (like a username/password). So we decided to stop supporting docker links (in a new pull request that will come tomorrow).
In the mean time, there's no need to add support for docker links for RabbitMQ, so it's back to the `BROKER_URL` environment variable (with the addition of a `CELERY_BROKER_URL` that gets prioritized over `BROKER_URL` for Celery)